### ISSUE TITLE
Handle WMS with "?" in url

### ISSUE THREAD

**Issue by ClaudioCas** [socib/Leaflet.TimeDimension#43]
Dear all,
I need to use the TimeDimension plugin for leaflet with a WMS server that has a special token in the request (var urlWMS = "https://myurl.com/wms?token=password";)

In Layer WMS i change those lines in _requestTimeDimensionFromCapabilities function:
Row 234
`  var wms = this._baseLayer.getURL();
   var url = wms + "?service=WMS&version=" +
    this._wmsVersion + "&request=GetCapabilities";`
with
`   var url = wms;
        if (wms.indexOf('?') > -1)
            url += '&'
        else
            ulr += '?'
        url += "service=WMS&version=" + this._wmsVersion + "&request=GetCapabilities";`

With this code, the url will generate correctly without the double '?'.
I hope you can add this modification in the main code.
Claudio

**Comment by ram-one** [comment]
Why don't you just add a parameter to your layer ?
```javascript
var testWMS = "http://thredds.socib.es/thredds/wms/observational/hf_radar/hf_radar_ibiza-scb_codarssproc001_aggregation/dep0001_hf-radar-ibiza_scb-codarssproc001_L1_agg.nc"
var testLayer = L.nonTiledLayer.wms(testWMS, {
    layers: 'sea_water_velocity',
    version: '1.3.0',
    format: 'image/png',
    transparent: true,
    attribution: 'SOCIB HF RADAR | sea_water_velocity',
    token: 'PASSWORD' // <-- Added parameter
});
var testTimeLayer = L.timeDimension.layer.wms(testLayer);
```
Every options that are not used are given to the WMS on URL query.

**Comment by ram-one** [comment]
@bielfrontera  I think this is closed by 2ce1882c91c13fe9c03fa3ace7e4b57123a187b8

**Comment by bielfrontera** [comment]
Yes, Is fixed!
